docs/source/pages/supported_technologies/index.rst

```diff
@@ -5,27 +5,55 @@
 Supported Technologies
 ======================
 
+-----------
+Build Tools
+-----------
+
+Macaron is able to detect the build and deployment scripts for the following build tools and package managers while analyzing the CI configurations,
+such as GitHub Actions workflows.
+
+* Maven
+* Gradle
+* Pip
+* Poetry
+* npm
+* Yarn
+* Go
+* Docker container
+
+
 ------------
 Git Services
 ------------
 
-.. list-table::
-   :header-rows: 1
+Currently, we support the following Git services for version control. If you need support for any other Git services, feel free to open a GitHub issue.
 
-   * - Git Service
-   * - `GitHub <https://github.com>`_
-   * - `GitLab <https://gitlab.com>`_
+* `GitHub <https://github.com>`_
+* `GitLab <https://gitlab.com>`_
 
 ------------
 CI Services
 ------------
 
+Currently, we support the following Continuous Integration (CI) services for automatically building and deploying artifacts. If you need support for any other CI services, feel free to open a GitHub issue.
+
 .. list-table::
    :header-rows: 1
 
    * - CI Service
+     - Support
    * - `GitHub Actions <https://github.com/features/actions>`_
-
+     -
+        * Detecting deployment steps by building a call graph for workflows and reachable shell scripts
+        * Support for various GitHub APIs, such as Releases
```

Comment:
I think it's not clear here what are the purposes of supporting various GitHub APIs (what are they used for?, which GitHub API entrypoints are supported).
Perhaps instead of saying that we support various GitHub APIs, we could say that we support verifying SLSA provenances published on GitHub Release instead?

Reply:
That's not correct and we do use APIs for purposes other than releases, such as checking workflow run status. I don't want to make this a long description and just indicating that we interact with GitHub is good enough.